python/extract_mappings.py

```diff
@@ -0,0 +1,306 @@
+#!/usr/bin/env python3
```

Comment:
Is this file used anywhere?

Reply:
No, it was used by mistake at first, in place of the `load_mappings.py` script, and I forgot to remove it. I'll do it in my next commit.